test/integration/crl_test.go

```diff
@@ -26,6 +27,9 @@ func runUpdater(t *testing.T, configFile string) {
 
 	c := exec.Command(binPath, "crl-updater", "-config", configFile, "-debug-addr", ":8022", "-runOnce")
 	out, err := c.CombinedOutput()
+	for _, line := range strings.Split(string(out), "\n") {
+		t.Log(line)
+	}
```

Comment:
Not seeing anything being validated here. Perhaps this a holdover from your own manual validation while writing this PR?
```suggestion
	for _, line := range strings.Split(string(out), "\n") {
		t.Log(line)
	}
```

Reply:
Yes, it's left over from my debugging. But it's also legitimately useful: it outputs nothing as long as the test is passing, and outputs the crl-updater logs (which would otherwise be completely swallowed, because it's not the instance spun up by start-servers) if the test fails. I'm going to leave it in, but add an explanatory comment.